src/Admin/Extensions/Extensions.php

```diff
@@ -355,6 +329,45 @@ static function ( $a, $b ) {
 			}
 		);
 
+		return $populated_extensions;
+	}
+
+	/**
+	 * Get the list of WPGraphQL extensions.
+	 *
+	 * @return PopulatedExtension[] The list of extensions.
+	 */
+	public function get_extensions(): array {
+		if ( ! isset( $this->extensions ) ) {
+			// @todo Replace with a call to the WPGraphQL server.
+			$extensions = Registry::get_extenions();
+
+			/**
+			 * Filter the list of extensions, allowing other plugins to add or remove extensions.
+			 *
+			 * @see Admin\Extensions\Registry::get_extensions() for the correct format of the extensions.
+			 *
+			 * @param array<string,mixed> $extensions The list of extensions.
+			 */
+			$extensions = apply_filters( 'graphql_get_extensions', $extensions );
```

Comment:
Since we're not doing anything expensive there's no need for the tech-debt of a second `pre_*` filter to short-circuit anything.